### ISSUE TITLE
Protocol: Add more supported IRCds

### ISSUE THREAD

**Issue by peerau** [peerau/A#3]
Due to the.. defunctness of Charybdis from the Atheme project, we need to move towards supporting other ircds, not %100 sure where we should focus now though, Suggestions?

**Comment by peerau** [comment]
Yes this also means the README is wrong - that was the Proof of Concept readme. lookin at you @staticfox

**Comment by staticfox** [comment]
InspIRCd

**Comment by peerau** [comment]
+1 Inspircd, also Unreal maybe?

**Comment by staticfox** [comment]
uinreal's about to release 4.0 so maybe

**Comment by CuleX** [comment]
Context: The plan originally was to port A to InspIRCd for Xertion. After my resignation there, that plan obviously died off.

Unless Unreal's protocol is close enough to TS6 (i.e., it has sane SIDs and UIDs), I suggest we *don't* support it. Unreal 3.2 doesn't have UIDs so it shouldn't receive any support whatsoever, 4.0 might.
